jobs/JGDAS_ATMOS_VERFRAD

```diff
@@ -6,82 +6,71 @@
 source "${HOMEgfs}/ush/preamble.sh"
 source "${HOMEgfs}/ush/jjob_header.sh" -e "verfrad" -c "base verfrad"
 
-export COMPONENT="atmos"
-
-export RADMON_SUFFIX=${RADMON_SUFFIX:-${RUN}}
-export CYCLE_INTERVAL=${CYCLE_INTERVAL:-6}
-
 ##############################################
 # Specify Execution Areas
 ##############################################
 export HOMEgfs=${HOMEgfs:-${NWROOT}/gfs.${gfs_ver}}
 export SCRgfs=${SCRgfs:-${HOMEgfs}/scripts}
 
-export FIXgdas=${FIXgdas:-${HOMEgfs}/fix/gdas}
+export FIXgdas=${FIXgdas:-${HOMEgfs}/fix/mon}
 export PARMmon=${PARMmon:-${HOMEgfs}/parm/mon}
 
 export HOMEradmon=${HOMEradmon:-${HOMEgfs}}
 export EXECradmon=${EXECradmon:-${HOMEradmon}/exec}
 export FIXradmon=${FIXradmon:-${FIXgfs}}
 export USHradmon=${USHradmon:-${HOMEradmon}/ush}
 
-
 ###################################
 # source the parm file
 ###################################
 parm_file=${parm_file:-${PARMmon}/da_mon.parm}
 .  ${parm_file}
 
-
 #############################################
 #  determine PDY and cyc for previous cycle
 #############################################
-
-pdate=$(${NDATE} -6 ${PDY}${cyc})
-echo "pdate = ${pdate}"
-
-export P_PDY=${pdate:0:8}
-export p_cyc=${pdate:8:2}
+GDATE=$(${NDATE} -"${assim_freq}" "${PDY}${cyc}")
+export gPDY=${GDATE:0:8}
+export gcyc=${GDATE:8:2}
 
 #############################################
 # COMOUT - WHERE GSI OUTPUT RESIDES
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
 YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_ANALYSIS
 YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_RADMON
-YMD=${P_PDY} HH=${p_cyc} generate_com -rx COM_ATMOS_RADMON_PREV:COM_ATMOS_RADMON_TMPL
+YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_RADMON_PREV:COM_ATMOS_RADMON_TMPL
 
 export TANKverf_rad=${TANKverf_rad:-${COM_ATMOS_RADMON}}
 export TANKverf_radM1=${TANKverf_radM1:-${COM_ATMOS_RADMON_PREV}}
 
 if [[ ! -d ${TANKverf_rad} ]]; then mkdir -p -m 775 ${TANKverf_rad} ; fi
 if [[ ! -d ${TANKverf_radM1} ]]; then mkdir -p -m 775 ${TANKverf_radM1} ; fi
 
-########################################
-# Set necessary environment variables
-########################################
-export RAD_AREA=${RAD_AREA:-glb}
-
+#############################################
+#  Filenames
 export biascr=${biascr:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.abias}
 export radstat=${radstat:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.radstat}
+export satype_file=${satype_file:-${FIXgdas}/gdas_radmon_satype.txt}
 
-echo " "
-echo "JOB HAS STARTED"
-echo " "
+#  Other variables
+export RAD_AREA=${RAD_AREA:-glb}
+export MAKE_CTL=${MAKE_CTL:-1}
+export MAKE_DATA=${MAKE_DATA:-1}
+export USE_ANL=${USE_ANL:-1}
+export PDATE=${PDY}${cyc}
+export DO_DIAG_RPT=${DO_DIAG_RPT:-1}
+export DO_DATA_RPT=${DO_DATA_RPT:-1}
```

Comment:
Except `PDATE`, these should be probably moved to a config file.

Reply:
Moving.